api/compression.md

```diff
@@ -11,14 +11,36 @@ policy](/compression/add_compression_policy/) for when to compress chunks.
 Advanced usage of compression allows users to [compress chunks
 manually](/compression/compress_chunk), instead of automatically as they age.
 
+Compressed chunks have the following limitations:
+- `ROW LEVEL SECURITY` is not supported.
+- Constraints are not fully supported:
+    - Unique constraints (`UNIQUE`) are restricted to compression `SEGMENTBY`
+      and `ORDER BY` columns.
+    - Primary Keys (`PRIMARY KEY`) are restricted to compression `SEGMENTBY`
+      and `ORDER BY` columns.
+    - Constraints are restricted to compression `SEGMENTBY` columns when
+    applicable. Otherwise they are unsupported.
+  
 ### Restrictions
 
-The current version does not support altering or inserting data into compressed
-chunks. The data can be queried without any modifications, however if you
-need to backfill or update data in a compressed chunk you will need to
+The current version does not fully support altering or modifying data in
+compressed chunks. The data can be queried without any modifications, however
+if you need to backfill or update data in a compressed chunk you will need to
```

Comment:
This confuses me. Backfill doesn't _require_ that users decompress first... correct? Could we maybe just be more straight forward:

```suggestion
As of TimescaleDB 2.3, users _can_ insert new data into compressed hypertable chunks. However, updating or deleting existing data in a compressed chunk still requires that you decompress the chunk, perform your data
modifications, and then compress the chunk again.
existing 
```

Reply:
I'm also torn now that I read the next section. It seems like we're repeating things a few times said different ways which is a little confusing to me. Is it better to maybe just put a table in here that says:

### Restrictions

In general, compressing a hypertable does impose some limitations on the types of data modifications that can occur on data inside a compressed chunk. The table below provides a brief outline of allowed modifications based on the version of TimescaleDB you are currently running.

| TimescaleDB Version | Supported data modifications on compressed chunks |
|----------------------|----------------------------|
|  1.6 - 2.0 | Data and schema modifications supported. |
| 2.1-2.2 | Schema may be modified on compressed hypertables. Data modification not supported |
| 2.3 | Schema modifications and basic insert of new data is allowed. Deleting, updating, and some advanced insert statements are not supported |


Thoughts @Loquacity ??

Reply:
I've added the table.